### ISSUE TITLE
[Training Issue]

### ISSUE THREAD

**Issue by xunkaixin** [eric612/MobileNet-YOLO#163]
Model :  yolov3

Backbone : efficientnet

Questions : different scale width large different loss (train from scratch)

Log files : 
I0909 14:54:48.172024 25166 solver.cpp:253] Iteration 418500 (143035 iter/s, 2.92586s/100 iters), loss = 3.71342
I0909 14:54:48.172063 25166 solver.cpp:272]     Train net output #0: det_loss1 = 0.146344 (* 1 = 0.146344 loss)
I0909 14:54:48.172068 25166 solver.cpp:272]     Train net output #1: det_loss2 = 0.919512 (* 1 = 0.919512 loss)
I0909 14:54:48.172072 25166 solver.cpp:272]     Train net output #2: det_loss3 = 2.481 (* 1 = 2.481 loss)

how to improve loss3 ?   yolo_data_jitter : 0.7
thank you

**Comment by ycxia** [comment]
@xunkaixin  Does caffe support efficientnet? can you release your prototxt? thank you!
